README.md

```diff
@@ -20,15 +20,23 @@ or the language specifier after them.
 ### Question 1
 
 Define a method called `offer_rose`, which should take one argument named `person`.
+// class Flower
+    def offer_rose
+      return "rose"
+      end
+  end
 
 When called the method should `puts` "Would you take this rose, `person`, in exchange for giving an old beggar woman shelter from the bitter cold?"
+// class Flower
+    def offer_rose
+      puts "Would you take this rose, `person`, in exchange for giving an old beggar woman shelter from the bitter cold?"
+      end
+  end
 
 Demonstrate calling the method, passing in "young prince" as the argument.
 
 Write your code here:
-```ruby
-# code here
-```
+youngPrince.offer_rose
```

Comment:
A couple of notes...

* "young prince" is a string, not a variable.
* "young prince" should be passed into `offer_rose` as an argument. It does not call the `offer_rose` method, as you have written.

```rb
offer_rose("young prince")
```